Makefile

```diff
@@ -36,9 +36,9 @@ COMMON_ARGS += --build-arg=GOFUMPT_VERSION=$(GOFUMPT_VERSION)
 COMMON_ARGS += --build-arg=TESTPKGS=$(TESTPKGS)
 TOOLCHAIN ?= docker.io/golang:1.15-alpine
 
-# help menu
+# Help Menu
 
-export define HELP_MENU_HEADER
+define HELP_MENU_HEADER
```

Comment:
this is auto-generated, please don't update it

Reply:
```
make lint
```
does not work :(
I've copy/past this from talos repo...

Reply:
are you on Mac? Or not using GNU make?

Reply:
yep, Mac OS )

```
make --version
GNU Make 3.81
Copyright (C) 2006  Free Software Foundation, Inc.
This is free software; see the source for copying conditions.
There is NO warranty; not even for MERCHANTABILITY or FITNESS FOR A
PARTICULAR PURPOSE.

This program built for i386-apple-darwin11.3.0
```

Reply:
I've revert Makefile...

Reply:
Should work if you update make maybe. I'm using 4.1 currently and it works